provider/README.md

```diff
@@ -127,8 +127,19 @@ represented as a GeoJSON [`Feature`](https://tools.ietf.org/html/rfc7946#section
 }
 ```
 
+#### Intersection Operation
+For the purposes of this specification, the intersection of two geographic datatypes is defined according to the [`ST_Intersects` PostGIS operation](https://postgis.net/docs/ST_Intersects.html)
+
+> If a geometry or geography shares any portion of space then they intersect. For geography -- tolerance is 0.00001 meters (so any points that are close are considered to intersect).<br>
+> Overlaps, Touches, Within all imply spatial intersection. If any of the aforementioned returns true, then the geometries also spatially intersect. Disjoint implies false for spatial intersection.
+
 [Top][toc]
 
+### Municipality Boundary
+Municipalities requiring MDS Provider API compliance should provide an unambiguous digital source for the municipality boundary. This boundary must be used when determining which data each `provider` API endpoint will include.
+
+The boundary should be defined as a polygon or collection of polygons. The file defining the boundary should be provided in Shapefile or GeoJSON format and hosted online at a published address that all providers and `provider` API consumers can access and download.
```

Comment:
Could we specify GeoJSON explicitly? Shapefile is generally not the best format to work with in my experience. Obviously, we could do either but I like specifying `geojson` as that seems to be what MDS and `mds-provider-services` use.

Reply:
I agree that Shapefile isn't as generally useful *directly*, but there are many tools to convert Shapefile -> geoJSON, and Shapefile is more of a de-facto standard format for many municipalities.

Reply:
makes sense!